### PR TITLE
@#444 | display tracing

### PR DESCRIPTION
**Pull request by datphan**
connect #444

### CODE REVIEW

File: lib/teracy-dev/logging.rb [hunk 1]
```diff
@@ -60,8 +60,9 @@ def self.configure_logger_for(classname)
             msg = TeracyDev::Common.yellow(msg)
           end
           # display log message here with tracing
-          # TODO: must display the trace of the exact line numeber having logger call (file, line number, function name)
-          tracing = TeracyDev::Common.light_gray("\t#{caller[5]}\n")
+          # display the trace of the exact line numeber having logger call (file, line number, function name)
+          # not display this logger tracing (< 3), but display 7 steps above
+          tracing = TeracyDev::Common.light_gray("\n#{caller[3..10] ? caller[3..10].join("\n") : caller}\n")
```
Comment (hoatle) [inline lib/teracy-dev/logging.rb]: we'd like to know exactly the line number of logger call, not the range.

**Review by hoatle — CHANGES_REQUESTED** [review]
not the range, but exactly line number of logger call

File: lib/teracy-dev/logging.rb [hunk 2]
```diff
@@ -60,8 +60,8 @@ def self.configure_logger_for(classname)
             msg = TeracyDev::Common.yellow(msg)
           end
           # display log message here with tracing
-          # TODO: must display the trace of the exact line numeber having logger call (file, line number, function name)
-          tracing = TeracyDev::Common.light_gray("\t#{caller[5]}\n")
+          # display the trace of the exact line numeber having logger call (file, line number, function name)
+          tracing = TeracyDev::Common.light_gray("\n#{caller[3]}\n")
```
Comment (datphan) [inline lib/teracy-dev/logging.rb:64]: @hoatle 
```
/Users/datphan/teracy/k8s-dev/extensions/teracy-dev-core/lib/teracy-dev-core/config/provisioners.rb:12:in `block in configure_node'
```
is this what you mean?

**Review by hoatle — APPROVED** [review]
LGTM

**Pull request merged**